specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-08-01-preview/examples/GetInsights.json

```
@@ -0,0 +1,147 @@
+                }
+              },
+              "justification": {
+                "expectedValue": 34,
+                "deltaPercent": -0.1,
+                "expectedMin": 32,
+                "expectedMax": 35,
+                "isAnomaly": false,
+                "currency": "USD",
+                "average": 32,
+                "minimum": 31,
+                "maximum": 33,
+                "actual": 31.5
+              }
+            }
+          },
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.CostManagement/insights/insightsId_345",
+            "name": "insightsId_345",
+            "type": "Microsoft.CostManagement/insights",
+            "kind": "NormalizedUsageAnomaly",
+            "location": null,
+            "sku": null,
+            "eTag": null,
+            "properties": {
+              "status": "New",
+              "eventDate": "2022-06-14T00:00:00",
+              "scope": "/subscriptions/00000000-0000-0000-0000-000000000000",
+              "impactTerm": "ongoing",
+              "impactResolution": "2022-06-16T12:59:59Z",
+              "timeDetected": "2022-06-16T12:59:59Z",
+              "severity": "None",
+              "details": {
+                "resourceGroups": {
+                  "topChanges": [
+                    {
+                      "changePercent": 0.55,
+                      "percentOfTotal": 0.083,
+                      "average": 12.59,
+                      "change": 0.05,
+                      "status": "existing",
+                      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resourcegroupexample"
+                    }
+                  ],
+                  "count": {
+                    "changePercent": 0.0,
+                    "average": 14.0,
+                    "change": 0.0,
+                    "removed": 0,
+                    "added": 0,
+                    "total": 14
+                  },
+                  "units": {
+                    "changePercent": 0.02,
+                    "existingPercent": 0.02,
+                    "removed": 0.0,
+                    "removedPercent": 0.0,
+                    "addedPercent": 0.0,
+                    "added": 0.0,
+                    "existing": 0.02
+                  }
+                },
+                "dateRange": {
+                  "period": "P60D",
+                  "start": "2022-04-16T00:00:00.0000000Z",
+                  "end": "2022-06-14T00:00:00.0000000Z"
+                }
+              },
+              "justification": {
+                "expectedValue": 53.69,
+                "deltaPercent": 0.1,
+                "expectedMin": 53.56,
+                "expectedMax": 53.89,
+                "isAnomaly": false,
+                "currency": "USD",
+                "average": 53.69,
+                "minimum": 48.88,
+                "maximum": 54.23,
+                "actual": 53.12
```

Adding a new property that's coming soon:
```suggestion
                "actual": 53.12,
                "numberOfDays": 60
```